RecoMuon/Navigation/src/MuonNavigationPrinter.cc

```diff
@@ -52,7 +55,10 @@ MuonNavigationPrinter::MuonNavigationPrinter(const MuonDetLayerGeometry * muonLa
   PRINT("MuonNavigationPrinter")  << "BACKWARD:" << std::endl;
 
   vector<const DetLayer*> backward;
-  if ( enableRPC ) backward = muonLayout->allBackwardLayers();
+  if ( enableCSC & enableGEM & enableRPC ) backward = muonLayout->allBackwardLayers();
+  else if ( enableCSC & enableGEM & !enableRPC ) backward = muonLayout->allCscGemBackwardLayers(); // CSC + GEM
+  else if ( !enableCSC & enableGEM & !enableRPC ) backward = muonLayout->backwardGEMLayers(); //GEM only
+  else if ( enableCSC & !enableGEM & !enableRPC ) backward = muonLayout->backwardCSCLayers(); //CSC only
   else backward = muonLayout->backwardCSCLayers();
```

Comment:
this looks wrong for the case of (enableCSC == true, enableGEM == false, enableRPC == true):
- the old code would give me allBackwardLayers, 
- the new code would give me backwardCSCLayers
I suspect this is the origin of the differences in current (run1/run2) reconstruction